### PR TITLE
Adding template for typescript electron and webpack

### PR DESCRIPTION
**Pull request by ryan-way**

### CODE REVIEW

File: src/routes/templates/templates.json [hunk 1]
```diff
@@ -776,5 +776,19 @@
...
+	},
+	{
+		"title": "Svelte-Typescript-Electron-Webpack",
+		"url": "https://github.com/ryan-way-boilerplate/stew",
+		"description": "Template with svelte typescript electron and webpack",
+		"npm": "",
+		"addedOn": "2022-03-21",
+		"category": "Svelte",
+		"tags": [
+			"typescript",
```
Comment (benmccann) [inline src/routes/templates/templates.json]: you need to run prettier. this is causing `lint` to fail

Comment (ryan-way) [reply src/routes/templates/templates.json]: updated

File: src/routes/templates/templates.json [hunk 2]
```diff
@@ -776,5 +776,15 @@
 		"category": "SvelteKit",
 		"tags": ["blog", "templates"],
 		"stars": 0
+	},
+	{
+		"title": "Svelte-Typescript-Electron-Webpack",
+		"url": "https://github.com/ryan-way-boilerplate/stew",
+		"description": "Template with svelte typescript electron and webpack",
```
Comment (benmccann) [inline src/routes/templates/templates.json]:
```suggestion
		"description": "Template with Svelte, Typescript, Electron, and webpack",
```

**Comment by benmccann** [comment]
this PR has a merge conflict and would need to be rebased

**Pull request merged**